### PR TITLE
Implement better reprs for Runnables

### PR DESCRIPTION
**Pull request by nfcampos**
```
ChatPromptTemplate(messages=[SystemMessagePromptTemplate(prompt=PromptTemplate(input_variables=[], template='You are a nice assistant.')), HumanMessagePromptTemplate(prompt=PromptTemplate(input_variables=['question'], template='{question}'))])
| RunnableLambda(lambda x: x)
| {
    chat: FakeListChatModel(responses=["i'm a chatbot"]),
    llm: FakeListLLM(responses=["i'm a textbot"])
  }
```

### CODE REVIEW

File: libs/langchain/tests/unit_tests/schema/runnable/test_runnable.py [hunk 1]
```diff
@@ -2275,6 +2284,7 @@ def _lambda(x: int) -> Union[int, Runnable]:
 
     output = await runnable.with_retry(
         stop_after_attempt=2,
+        wait_exponential_jitter=False,
```
Comment (hinthornw) [inline libs/langchain/tests/unit_tests/schema/runnable/test_runnable.py:2287]: Speed up tests?

Comment (nfcampos) [reply libs/langchain/tests/unit_tests/schema/runnable/test_runnable.py:2287]: Yep

File: libs/langchain/langchain/schema/runnable/utils.py [hunk 2]
```diff
@@ -87,12 +87,38 @@ def visit_AsyncFunctionDef(self, node: ast.AsyncFunctionDef) -> Any:
...
+        code = inspect.getsource(func)
+        tree = ast.parse(textwrap.dedent(code))
+        visitor = GetLambdaSource()
+        visitor.visit(tree)
+        return visitor.source
+    except (SyntaxError, TypeError, OSError):
         return None
+
+
+def indent_lines_after_first(text: str, prefix: str) -> str:
```
Comment (eyurtsev) [inline libs/langchain/langchain/schema/runnable/utils.py:128]: a bit odd to get a prefix here vs. n_spaces

Comment (nfcampos) [reply libs/langchain/langchain/schema/runnable/utils.py:128]: Similar api to textwrap.indent

**Comment by eyurtsev** [comment]
sorry i'm comically bad with the linters

**Pull request merged**